pkg/restapi/rp/operation/testdata/context/consent-credential-v1.jsonld

```diff
@@ -5,45 +5,32 @@
       "name":"http://schema.org/name",
```

Comment:
there's another copy of this file in the BDD tests folder

Reply:
@llorllale [done](https://github.com/trustbloc/edge-adapter/compare/e6c1f198f689cad99da478f8e1ef9edd349f4c8e..8eeb9c4be51ebc0319672eca4cb4ae432d1bfb3b).